src/dotnet/devcontainer-feature.json

```diff
@@ -1,35 +1,31 @@
 {
     "id": "dotnet",
-    "version": "1.1.4",
+    "version": "2.0.0",
     "name": "Dotnet CLI",
     "documentationURL": "https://github.com/devcontainers/features/tree/main/src/dotnet",
-    "description": "Installs the .NET CLI. Provides option of installing sdk or runtime, and option of versions to install. Uses latest version of .NET sdk as defaults to install.",
+    "description": "Installs .NET SDK, which includes the .NET CLI and the shared runtime. Offers options to install multiple SDK versions.",
     "options": {
         "version": {
             "type": "string",
             "proposals": [
                 "latest",
-                "7",
-                "6",
-                "3.1"
+                "lts",
+                "8.0",
+                "7.0",
+                "6.0"
             ],
             "default": "latest",
-            "description": "Select or enter a dotnet CLI version. (Available versions may vary by Linux distribution.)"
+            "description": "Version of .NET SDK to install. Use 'latest' for the latest version, 'lts' for the latest LTS version, 'X.Y' or 'X.Y.Z' for a specific version."
         },
-        "runtimeOnly": {
-            "type": "boolean",
-            "default": false,
-            "description": "Install just the dotnet runtime if true, and sdk if false."
-        },
-        "installUsingApt": {
-            "type": "boolean",
-            "default": true,
-            "description": "If true, it installs using apt instead of the release URL"
+        "additionalVersions": {
+            "type": "string",
+            "default": "",
+            "description": "Comma-separated list of additional .NET SDK versions to install. Use 'latest' for the latest version, 'lts' for the latest LTS version, 'X.Y' or 'X.Y.Z' for a specific version."
         }
     },
     "containerEnv": {
-        "DOTNET_ROOT": "/usr/local/dotnet/current",
-        "PATH": "/usr/local/dotnet/current:${PATH}"
+        "DOTNET_ROOT": "/usr/share/dotnet",
+        "PATH": "${DOTNET_ROOT}:${PATH}"
```

Comment:
Specifying PATH here might be unnecessary, the install script already does this unless `--no-path` is used.

I am not sure if it is better to remove this config, or keep it and run the script with `--no-path`

![image](https://github.com/devcontainers/features/assets/1583241/65d774b0-5403-476d-a6ed-787a5ada8164)

Reply:
Removing PATH from containerEnv breaks the test 🤔

```
dotnet_helpers.sh: line 28: dotnet: command not found
```

Reply:
Oh because the script only exports it for the current session... so I guess better to use `--no-path`

Note: also need to add `${DOTNET_ROOT}/tools` to PATH for global tools.

Reply:
Fixed in 2d0fb73697461a28b2628b69962263043d6873dd